ansible/condor/roles/de-image-janitor/templates/excludes.j2

```diff
@@ -1,24 +1 @@
-{{ docker.registry.base }}/backwards-compat:latest
-{{ docker.registry.base }}/backwards-compat:{{docker.tag}}
-discoenv/porklock:{{docker.tag}}
-discoenv/porklock:latest
-{{ docker.registry.base }}/iplant_data:latest
-gliderlabs/consul-server:latest
-{{ docker.registry.base }}/de-configs-{{environment_name}}:{{docker.tag}}
-discoenv/curl-wrapper:latest
-{{ docker.registry.base }}/jupyter-lab:1.0
-{{ docker.registry.base }}/shiny:1.0.0
-{{ docker.registry.base }}/rstudio-nginx:3.5.0
-sblsdsc/mmtf-pyspark-vice-0.3.6:16
-sblsdsc/mmtf-genomics-vice-0.3.6:8
-discoenv/cas-proxy:{{docker.tag}}
-discoenv/cas-proxy:latest
-cyverse/jupyterlab-nanodj:latest
-{{ docker.registry.base }}/jupyterlabqiime2:2018.11.2
-{{ docker.registry.base }}/genomicseducationalliance-jupyterlab-rnaseq-qc:dev_2.1
-{{ docker.registry.base }}/quaternionclassifier:latest
-{{ docker.registry.base }}/powerstateclassifier:latest
-{{ docker.registry.base }}/behaviorclassifier:latest
-{{ docker.registry.base }}/turboprop:latest
-genomicseducationalliance/rna-seq-leptin:dev_0.2
-cyversevice/rstudio-geospatial:terraref-3.6.0
+{{porklock_image}}:{{porklock_tag}}
```

Comment:
Most of the old exclusions were outdated, so I removed all of them except for this one.

Reply:
The only other one I could think of would be the url-import tool, but That'll also be quite small and I dunno how much it's used. So porklock is probably sufficient.

Reply:
That's a good point. I'll add it to the list.